docs/k8s-helm.md

```diff
@@ -61,12 +61,17 @@ timeplusd:
     stream:
       className: <Your storage class name>
       size: 100Gi
+      # Keep this to be `null` if you are on Amazon EKS with EBS CSI controller. Otherwise please carefully check your 
```

Comment:
since it's long and you already break to 2 lines, please break before 'Otherwise'

Reply:
updated